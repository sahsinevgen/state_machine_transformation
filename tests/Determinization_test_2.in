3
1
1
3

4
1 2 a
2 1 a
1 3 #
2 3 b