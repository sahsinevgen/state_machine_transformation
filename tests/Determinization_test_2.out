4
1
2
1 4 
8
1 2 a
1 3 b
2 1 a
2 4 b
3 3 a
3 3 b
4 3 a
4 3 b